test-requirements.txt

```diff
@@ -1,3 +1,3 @@
 # Use latest so that layer-basic tests against main branch
-git+https://github.com/juju/charm-tools.git#egg=charm-tools
```

Comment:
Need to bring this back once we land https://github.com/juju/charm-tools/pull/620